### PR TITLE
tests: add bionic system to google backend

### PR DESCRIPTION
**Pull request by sergiocazzolato**

### CODE REVIEW

**Comment by mvo5** [comment]
Yay, thanks Sergio!

File: tests/upgrade/basic/task.yaml [hunk 1]
```diff
@@ -18,7 +18,10 @@ execute: |
     . "$TESTSLIB/snaps.sh"
 
     echo "Remove snapd and snap-confine"
-    distro_purge_package snapd snapd-selinux snap-confine || true
+    distro_purge_package snapd snap-confine || true
+    if [[ "$SPREAD_SYSTEM" = fedora-* ]] ; then
+        distro_purge_package snapd-selinux || true
```
Comment (zyga) [inline tests/upgrade/basic/task.yaml]: Nice! Thank you!

**Comment by bboozzoo** [comment]
Unit tests failed, but the log does not contain the test that failed. Restarted the build and we'll see if this repeats again.

**Comment by sergiocazzolato** [comment]
@bboozzoo I was because of a change in the behaviour of 'go test'. It is not creating anymore the  .coverprofile in case there is no test. @chipaca has pushed a fix for that.

**Comment by bboozzoo** [comment]
squashfs needs an update?
```
=== RUN   Test

----------------------------------------------------------------------
FAIL: squashfs_test.go:356: SquashfsTestSuite.TestBuildDate

squashfs_test.go:369:
    c.Check(math.Abs(now.Sub(snap.BuildDate()).Seconds()) <= 61, Equals, true)
... obtained bool = false
... expected bool = true

OOPS: 20 passed, 1 FAILED
--- FAIL: Test (0.09s)
```

**Comment by sergiocazzolato** [comment]
@bboozzoo well, yes, I need to enable propoed on that image to make that work.

**Pull request merged**